lua/tests/fixtures/example-lean3-project/test.lean

```diff
@@ -1,5 +1,7 @@
 import test.test1
 
+set_option pp.notation false
```

Comment:
In some runs I noticed Lean 3 unexpectedly dropping notation, causing the test to fail, so I thought it was best to normalize this (haven't seen an issue with Lean 4 yet).